azure-pipelines-rpi: Upload .dtb files also

romandariana opened the pull request:
## PR Description

There are also some .dtb files in the overlays folder. Modify *.dtbo to *.dtb* in order to include these files.

## PR Type
- [ ] Bug fix (a change that fixes an issue)
- [ ] New feature (a change that adds new functionality)
- [ ] Breaking change (a change that affects other repos or cause CIs to fail)

## PR Checklist
- [x] I have conducted a self-review of my own code changes
- [ ] I have tested the changes on the relevant hardware
- [ ] I have updated the documentation outside this repo accordingly (if there is the case)

AAndrisa commented:
Maybe you could modify also this `if` conditions to include .dtb files from the overlays directory
[https://github.com/analogdevicesinc/linux/blob/0019c28fd808bd9873504c861a602b90032e2216/ci/travis/prepare_artifacts_rpi.sh#L43]

romandariana commented:
v2 -> v3:
    - split into 2 commits, 1 for each file. i added the script change as the second commit since if I understood correctly that is only for validation (?)
